pkg/runtime/wfengine/workflow.go

```diff
@@ -282,6 +288,7 @@ func (wf *workflowActor) addWorkflowEvent(ctx context.Context, actorID string, h
 	if err != nil {
 		return err
 	}
+	wfLogger.Debugf("workflow actor '%s': adding event %v to the workflow inbox", actorID, e)
```

Comment:
Should we surround the event in '' like we do with other values we put in the log?

Reply:
Done